src/main/resources/openapi/schemas/titleDataEntry.json

```diff
@@ -1,26 +1,30 @@
 {
-  "description": "title data entry",
+  "description": "title data entry from counter reports",
   "type": "object",
   "properties": {
     "id": {
       "type": "string",
       "description": "title data identifier"
     },
-    "reportTitleId": {
+    "titleEntryId": {
       "type": "string",
-      "description": "title identifier"
+      "description": "key to title entries"
     },
     "counterReportId": {
       "type": "string",
       "description": "counter report identifier"
     },
+    "counterReportTitle": {
```

Comment:
Do we need counterReportTitle here if we decided to keep it in the TITLE_ENTRY?

Reply:
Not really. Do you expect the UI to do the lookup, then? That would mean more work tvere